pymongo/collection.py

```diff
@@ -2280,6 +2279,178 @@ def index_information(
             info[index.pop("name")] = index
         return info
 
+    def list_search_indexes(
+        self,
+        name: Optional[str] = None,
+        session: Optional["ClientSession"] = None,
+        comment: Optional[Any] = None,
+        **kwargs: Any,
+    ) -> CommandCursor[_DocumentType]:
+        """Return a cursor over search indexes for the current collection.
+
+        :Parameters:
+            `name` (optional) - If given, the name of the index to search
```

Comment:
Can name be a regex or a glob or has to be an exact match?

Reply:
Exact match